deploy-service/common/src/main/resources/sql/deploy.sql

```diff
@@ -24,26 +23,32 @@ CREATE TABLE IF NOT EXISTS environs (
     last_update   BIGINT              NOT NULL,
     accept_type   VARCHAR(32)         NOT NULL,
     email_recipients VARCHAR(1024),
-    notify_authors   TINYINT(1),
+    notify_authors   TINYINT(1)       NOT NULL DEFAULT 0,
     watch_recipients VARCHAR(1024),
     metrics_config_id VARCHAR(22),
     alarm_config_id     VARCHAR(22),
     webhooks_config_id  VARCHAR(22),
-    max_deploy_num      INT           NOT NULL,
-    max_deploy_day      INT           NOT NULL,
+    max_deploy_num      INT           NOT NULL DEFAULT 5000,
+    max_deploy_day      INT           NOT NULL DEFAULT 365,
     is_docker           TINYINT(1)    NOT NULL DEFAULT 0,
     max_parallel_pct    TINYINT(1)    NOT NULL DEFAULT 0,
     state               VARCHAR(32)   NOT NULL,
+    system_priority INT,
     cluster_name        VARCHAR(128),
     max_parallel_rp     INT           NOT NULL DEFAULT 1,
-    override_policy     VARCHAR(32)   NOT NULL,
+    override_policy     VARCHAR(32)   NOT NULL DEFAULT 'OVERRIDE',
     schedule_id         VARCHAR(22),
     deploy_constraint_id      VARCHAR(22),
     external_id          CHAR(36),
     allow_private_build TINYINT(1)    NOT NULL DEFAULT 0,
     ensure_trusted_build TINYINT(1)    NOT NULL DEFAULT 0,
-    PRIMARY KEY   (env_id)
-) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+    stage_type          VARCHAR(32)   NOT NULL DEFAULT 'DEFAULT',
+    is_sox              TINYINT(1)    NOT NULL DEFAULT 0,
+    PRIMARY KEY   (env_id),
+    UNIQUE KEY sc_config_id (sc_config_id),
```

Comment:
hi @liyaqin1 are these changes missing in migrations?

Reply:
Btw - migrations are here... ideally these would live in the same place in the future:

https://github.com/pinterest/teletraan/tree/2110c7c987faf5336a917781480da34c135494c4/tools/mysql

Reply:
We don't want to update this file, instead we should apply migrations before running the tests